acquia_lift.admin.inc

```diff
@@ -183,6 +183,15 @@ function acquia_lift_admin_form($form, &$form_state) {
     '#description' => t('The number of variations selected by Acquia Lift for display to visitors before the campaign can be completed. Applies only to campaigns with the "End when campaign thresholds are reached" option selected.'),
     '#element_validate' => array('element_validate_number'),
   );
+  $form['acquia_lift_report_max_days'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Maximum days of reporting history'),
+    '#size' => 3,
+    '#required' => TRUE,
+    '#default_value' => variable_get('acquia_lift_report_max_days', 14),
+    '#description' => t('The maximum number of days to show reporting history for.'),
+    '#element_validate' => array('element_validate_number'),
```

Comment:
Maybe element_validate_integer_positive instead?  I think element_validate_number will still allow 0 and negatives, etc.